_includes/pbc/all/install-features/202311.0/unified-commerce/install-the-warehouse-picking-feature.md

```diff
@@ -108,6 +132,24 @@ Make sure the following changes have been triggered in transfer objects:
 | Stock.pickingListStrategy                      | property | created | src/Generated/Shared/Transfer/StockTransfer                                   |
 | PushNotificationCollectionRequest.pickingLists | property | created | src/Generated/Shared/Transfer/PushNotificationCollectionRequestTransfer       |
 
+
+Optional: In case the Fulfillment Apps's early access OAuth Authorization was installed.
+
+Make sure the following changes have been applied by checking your database:
+
+| DATABASE ENTITY               | TYPE  | EVENT   |
+|-------------------------------|-------|---------|
+| spy_oauth_code_flow_auth_code | table | created |
+
+
+Make sure the following changes have been triggered in transfer objects:
+
+| TRANSFER                   | TYPE  | EVENT   | PATH                                                             |
+|----------------------------|-------|---------|------------------------------------------------------------------|
+| AuthCode                   | class | created | src/Generated/Shared/Transfer/AuthCodeTransfer                   |
+| AuthCodeAttributes         | class | created | src/Generated/Shared/Transfer/AuthCodeAttributesTransfer         |
+| AuthCodeResponseAttributes | class | created | src/Generated/Shared/Transfer/AuthCodeResponseAttributesTransfer |
```

Comment:
Minor: there are a lot of transfer properties existing in OauthCodeFlow only as OauthRequest.code, OauthRequest.codeChallengeMethod, OauthRequest.codeVerifier, OauthRequest.codeChallenge, OauthRequest.state, OauthResponse.state, OauthResponse.code, etc. Please add only if you think it makes sense.